contracts/token/ERC1155/ERC1155.sol

```diff
@@ -14,7 +14,7 @@ import {IERC1155Errors} from "../../interfaces/draft-IERC6093.sol";
 /**
  * @dev Implementation of the basic standard multi-token.
  * See https://eips.ethereum.org/EIPS/eip-1155
- * Originally based on code by Enjin: https://github.com/enjin/erc-1155
+ * Originally based on code by Enjin: https://github.com/enjin/erc-1155/tree/9f6821af0f8544167a1e6e0649e56dc39b1d51c7
```

Comment:
This line was written in https://github.com/OpenZeppelin/openzeppelin-contracts/pull/1803/files, which was merged almost 5 years ago. It feels wrong changing that now, particularly referencing a commit from feb. 2023

Reply:
I agree with not changing it. I wouldn't add the newest found commit unless we find the correct commit the implementation was based on.